R/SSexecutivesummary.r

```diff
@@ -70,17 +87,14 @@ SSexecutivesummary <- function(replist,
                                format = TRUE,
                                match_digits = FALSE,
                                verbose = TRUE) {
-  # Make sure table.dir contains the report file
-  if (is.null(replist)) {
-    stop("The input 'replist' should refer to an R object created by the function 'SS_output'.")
-  }
-
-  if (plotfolder == "default") {
-    csv.dir <- paste0(replist[["inputs"]][["dir"]], "/tables")
-  }
-  if (plotfolder != "default") {
-    csv.dir <- paste0(plotfolder, "/tables")
-  }
+  csv.dir <- file.path(
+    ifelse(
+      plotfolder == "default",
```

Comment:
Additional thoughts on this: more flexibility in folder names not equal to "tables" (however implemented) would facilitate the creation of separate sets of tables with `format = TRUE` and `format = FALSE`. However, more user flexibility would mean users had to specify the full path to the tables and not the model folder in `sa4ss::es_table_tex()` due to the use of "tables" in https://github.com/pfmc-assessments/sa4ss/blob/ba6a480e6dfdfc208e8ba42510c90bb36ebae30c/R/es_table_tex.R#L53-L55.

Reply:
I would honestly like to get rid of the format option where the code just assumes `format = FALSE` and only exports the csv files. Then we can handle the formatting via sa4ss. I am guessing that I was the one who originally asked Chantel to output them both ways but I have since changed my mind in this instance.

Regarding how the directory is named, it does not really matter for sa4ss because the user has to tell the framework where to look for the tables. So, it would just be a breaking change for old assessments to be able to use the newest version of sa4ss, which I am fine with.

Reply:
When I developed this code originally there were fewer sa4ss (or markdown) users in the team so this was an attempt to provide something usable for everyone.  I think removing the formatting option would be fine since the few individuals who are still using Word are likely very comfortable doing a bit of user-specific formatting in Exel and it would greatly streamline the code making ongoing maintenance easier.